src/auth-manager/auth-manager.ts

```diff
@@ -347,7 +347,7 @@ export default class AuthManager<IUser, ISignInParams, IsSignedIn extends boolea
 
       await this._forceRefreshToken();
     } catch (error) {
-      if (axios.isAxiosError(error) && isClientErrorStatusCode(error.response?.status)) {
+      if (isAxiosError(error) && isClientErrorStatusCode(error.response?.status)) {
         await this._forceSignOut();
       }
```

Comment:
**Suggestion:** Consider adding a null check for `error.response` before using it in the `isAxiosError` condition. This will prevent potential `undefined` errors.
```suggestion
      if (isAxiosError(error) && error.response && isClientErrorStatusCode(error.response.status)) {
        await this._forceSignOut();
      }
```